src/Backend/Modules/ContentBlocks/ContentBlock/ContentBlockRepository.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Backend\Modules\ContentBlocks\ContentBlock;
+
+use Backend\Core\Language\LanguageName;
+use Doctrine\ORM\EntityRepository;
+use Doctrine\ORM\Query;
+
+class ContentBlockRepository extends EntityRepository
+{
+    /**
+     * @param LanguageName $language
+     *
+     * @return Query
+     */
+    public function getDataGridQuery(LanguageName $language)
+    {
+        return $this->getEntityManager()
+            ->createQueryBuilder()
+            ->select('cb.id, cb.title, cb.isHidden')
+            ->from(ContentBlock::class, 'cb')
+            ->where('cb.status = ?1 AND cb.language = ?2')
```

Comment:
I think there was a reason I didn't do that, but I'll look into it.
At the moment it is a bit of a hack to get doctrine to work with the datagrid, I might just make it less hacky :)